tests/performance-test/deploy/performance-test-job.yml

```diff
@@ -0,0 +1,52 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: saf-performance-test
+spec:
+  parallelism: 1
+  completions: 1
+spec:
+  template:
+    metadata:
+      label:
+        name: saf-performance-test
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: performance-test
+          image: 172.30.1.1:5000/sa-telemetry/performance-test:dev
```

Comment:
Enhancement here (future) would be to take these files and make them Ansible templates so that it could generate the registry dynamically like we do prior to the use of `deploy.sh`. Not necessary now, but something that would move this out of the minishift environment.

Don't do that now obviously. Future work.